src/navigate-pop/launching-applications.md

```diff
@@ -0,0 +1,92 @@
+<!--Sources
+Pop Training Guide (Google doc)
+https://support.system76.com/articles/pop-basics/
+
+-->
+# Locate, Launch, and Navigate Between Applications
+
+---
+
+## Browsing Installed Applications
+
+Installed applications can be viewed in the Applications menu and the Pop!\_Shop Installed screen.
+
+### The Applications Menu
+
+The Applications menu can be accessed from the Dock or by clicking `Applications` in the upper left corner of the screen.
+
+![Launch Applications Menu](/images/locate-launch-applications/launch-applications-menus.png)
+
+Applications can be grouped into folders. Click `Create Folder` and give the folder a name.
+
+![Create Application Folder](/images/locate-launch-applications/create-application-folder.png)
+
+Click and drag an application icon to add it to your folder.
+
+<video autoplay loop>
+    <source src="/images/locate-launch-applications/drag-application.webm" />
+</video>
+
+### The Pop!\_Shop
+
+Launch the Pop!\_Shop, then click the `Installed` button.
+
+![Pop Shop Installed Apps](/images/locate-launch-applications/pop-shop-installed-apps.png)
+
+## Launching Applications
+
+Applications can be launched from the Dock, using the Launcher, the Applications menu, and from the Terminal.
+
+### Using the Launcher
+
+Press the `SUPER` key or the Show Launcher icon to bring up the Launcher.
+
+![Show Launcher](/images/locate-launch-applications/show-launcher.png)
+
+Search for the name of the application, then hit `Enter`.
+
+![Search App Name](/images/locate-launch-applications/search-app.png)
+
+### Using the Applications Menu
+
+Click the Applications menu icon located in the Dock.
+
+![Click Applications Dock](/images/locate-launch-applications/click-applications-dock.png)
+
+Click the application's icon to launch it.
+
+![Click App Icon](/images/locate-launch-applications/click-app-icon.png)
+
+### Using the Dock
+
+Click the application's icon pinned to the Dock.
+
+![Dock Icon](/images/locate-launch-applications/dock-icon.png)
+
+You can pin additional applications to the Dock by right-clicking the application's icon in the Application menu and selecting `Add to Favorites`.
+
+![Add to Favorites](/images/locate-launch-applications/add-to-favorites.png)
+
+Remove applications from the Dock by right-clicking the application and selecting `Remove from Favorites`.
+
+![Remove from Favorites](/images/locate-launch-applications/remove-from-favorites.png)
+
+### Using Alt + F2
+
+Press `Alt` + `F2` to bring up a command entry window. Enter the name of the application and press `Enter`.
+
+![Command Launcher](/images/locate-launch-applications/command-launcher.png)
+
+### Using the Terminal
+
+Launch the Terminal from the dock; then type the name of the application hit `Enter`. You can also launch the Terminal by pressing `SUPER` + `T`.
+
+![Launch From Terminal](/images/locate-launch-applications/launch-from-terminal.png)
```

Comment:
Alt-F2 and the terminal won't take the name of the app in any form, it needs to be the name of the executable for the app. For example, the Pop!_Shop is going to require `io.elementary.appcenter`. Even if you explained that the app name usually needs to be lowercase like in Firefox's case, that wouldn't cover the Pop!_Shop case. Thoughts?

Reply:
I agree it could be misleading as presented. I'll clarify that the application's executable name must be used with these options.